### PR TITLE
Moltres edits

### PR DESCRIPTION
**Pull request by katyhuff**
These are mostly edits that do the following minor grammatical things (suggested by this MOOC I'm taking http://online.stanford.edu/course/writing-sciences-self-paced-spring-2016)

- [x] clunky nouns -> spunky verbs (progression->progress, expression->express)
- [x] reduce vague words (important, methodologic)
- [x] get rid of unnecessary prepositional phrases -- author clearing throat (It can be shown that)
- [x] get rid of extraneous adverbs (very, really, quite, basically, generally)
- [x] get rid of there are / there is
- [x] turn negatives to positives (she was not often right -> she was usually wrong)
- [x] get rid of extraneous prepositions (the meeting happened on monday -> the meeting happened monday) (they agreed that it was true -> they agreed it was true)
- [x] get rid of passive voice (is/was/are/were/be/been/am + past tense verb), replace with active voice (yes, it's ok to use first person in journal articles. [Some journals specifically request it in their style guide.](http://www.sciencemag.org/site/feature/contribinfo/prep/res/style.xhtml) )

### CODE REVIEW

File: Moltres.tex [hunk 1]
```diff
@@ -95,37 +95,37 @@ \section{Introduction}
 the inclusion of the \gls{MSR} among the Generation-IV reactor designs
 \cite{noauthor_gif_2015}, this reactor concept has gained renewed research
 interest in the past decade, with many new nuclear companies proposing both
-fluid-fuelled and solid-fuelled commerical \gls{MSR} designs
+fluid-fuelled and solid-fuelled commercial \gls{MSR} designs
 \cite{hyde_liquid_2015,leblanc_integral_2015,noauthor_-able_2017,scarlat_design_2014,noauthor_neutronics_2016}. Two
 key advantages offered by the fluid-fuelled \gls{MSR} are improved fuel
 utilization and no radiation damage constraint on attainable fuel
 burn-up. Together, these attributes result in significantly reduced core fissile
 inventories and spent nuclear fuel mass \cite{noauthor_gen_2008}.  These
```
Comment (lindsayad) [inline Moltres.tex]: Delete `These` at the end of this line

File: Moltres.tex [hunk 2]
```diff
@@ -95,37 +95,37 @@ \section{Introduction}
...
-characteristics give the \gls{MSR} a potentially unique capability for actinide
-burning and extension of fuel resources.
+Due to these characteristics give the \gls{MSR} potentially unique
```
Comment (lindsayad) [inline Moltres.tex]: Odd sentence

File: Moltres.tex [hunk 3]
```diff
@@ -468,10 +468,10 @@ \subsection{Comparison with \gls{MSRE}}
 profiles with cosinusoidal gamma heating and \gls{MSRE} design models
 \cite{briggs_molten-salt_1964} in the hottest channel and adjacent graphite. The
 profiles are in good agreement. In the \gls{MSRE} design, the moderator
-temperature at the reactor inlet is about 20 degrees Faerenheit larger than the
+temperature at the reactor inlet is about 20 degrees Fahrenheit larger than the
 fuel temperature, whereas the temperatures are about the same in the Moltres
-model. This difference can perhaps be attributed to the neglection of axial
-conduction in the \gls{MSRE} design model \cite[p. 99]{briggs_molten-salt_1964}.
+model. This difference is likely because axial the \gls{MSRE} design model
```
Comment (lindsayad) [inline Moltres.tex]: remove `axial`

File: Moltres.tex [hunk 4]
```diff
@@ -501,9 +501,9 @@ \subsection{Comparison with \gls{MSRE}}
 calculations \cite[p. 92]{briggs_molten-salt_1964} as shown in
 \cref{fig:msre_radial_fluxes}. The peak fast to thermal flux ratio is
 approximately 3.5 in the \gls{MSRE} design calculation as opposed to a ratio of
-3 for the Moltres calculation. The depression in the thermal flux in
-\cref{fig:msre_radial_fluxes} is caused by control rod thimbles and an extra
-volume of surrounding fuel that is not included in the Moltres calculations.
+3 for the Moltres calculation. Meanwhile, control rod thimbles and
```
Comment (lindsayad) [inline Moltres.tex]: Maybe remove `Meanwhile,`?

File: Moltres.tex [hunk 5]
```diff
@@ -528,17 +528,17 @@ \subsection{Comparison with \gls{MSRE}}
 \end{figure}
 
 \Cref{fig:axial_fluxes_compare} compares the axial flux profiles calculated by
-Moltres and the \gls{ORNL} \gls{MSRE} design model. Note that the radius for the
+Moltres and the \gls{ORNL} \gls{MSRE} design model. The radius for the
```
Comment (lindsayad) [inline Moltres.tex]: This was my mistake...change radius to radii

**Review by lindsayad — CHANGES_REQUESTED** [review]
Thanks for the changes! Just a couple of questions/suggestions

**Comment by katyhuff** [comment]
Fixed all suggested changes! Thanks for catching those!

**Review by lindsayad — APPROVED** [review]

**Pull request merged**